src/ert/_c_wrappers/enkf/hook_workflow.py

```diff
@@ -36,3 +36,6 @@ def __ne__(self, other):
 
     def __repr__(self):
         return f"HookWorkflow({self.getWorkflow().src_file}, {self.getRunMode()})"
+
+    def free(self):
```

Comment:
I've gone over the other implementations and also came to the conclusion that only these three are missing free() override. As far as i understand, using pass is also the way to go here, since we have no underlying content to actually free/release. :+1:

Reply:
right. one of them was an actual free, the hook workflow, but that failed a bunch of tests with ghastly "you're trying to break the universe" kind of errors, so i removed it ^^ i should have read the code more carefully before just adding that one, i guess :sweat_smile: